rskj-core/src/test/java/co/rsk/peg/federation/FederationSupportImplTest.java

```diff
@@ -2309,27 +2309,60 @@ void voteFederationChange_whenCreateIsCalledBy2DifferentAuthorizer_returnsSucces
-            Transaction secondAuthorizerTx = TransactionUtils.getTransactionFromCaller(signatureCache, FederationChangeCaller.SECOND_AUTHORIZED.getRskAddress());
-            ABICallSpec abiCallSpec = new ABICallSpec(FederationChangeFunction.CREATE.getKey(), new byte[][]{});
+            Transaction firstAuthorizerTx = TransactionUtils.getTransactionFromCaller(
+                signatureCache, FederationChangeCaller.FIRST_AUTHORIZED.getRskAddress());
+            Transaction secondAuthorizerTx = TransactionUtils.getTransactionFromCaller(
+                signatureCache, FederationChangeCaller.SECOND_AUTHORIZED.getRskAddress());
+            ABICallSpec abiCallSpec = new ABICallSpec(FederationChangeFunction.CREATE.getKey(),
+                new byte[][]{});
 
             // Act
 
             // First create call
-            int resultFromFirstAuthorizer = federationSupport.voteFederationChange(firstAuthorizerTx, abiCallSpec, signatureCache, bridgeEventLogger);
+            int resultFromFirstAuthorizer = federationSupport.voteFederationChange(
+                firstAuthorizerTx, abiCallSpec, signatureCache, bridgeEventLogger);
 
             // Second create call
-            int resultFromSecondAuthorizer = federationSupport.voteFederationChange(secondAuthorizerTx, abiCallSpec, signatureCache, bridgeEventLogger);
+            int resultFromSecondAuthorizer = federationSupport.voteFederationChange(
+                secondAuthorizerTx, abiCallSpec, signatureCache, bridgeEventLogger);
 
             // Assert
 
             assertEquals(FederationChangeResponseCode.SUCCESSFUL.getCode(), resultFromFirstAuthorizer);
             assertEquals(FederationChangeResponseCode.SUCCESSFUL.getCode(), resultFromSecondAuthorizer);
+
             assertThat(federationSupport.getPendingFederationSize(), is(0));
             assertNotNull(federationSupport.getPendingFederationHash());
 
         }
 
+        @Test
+        void voteFederationChange_commitEmptyFederation_returnsInsufficientMembersResponseCode() {
+
+            // Arrange
+
+            Transaction tx = TransactionUtils.getTransactionFromCaller(signatureCache, FederationChangeCaller.FIRST_AUTHORIZED.getRskAddress());
+            Transaction tx2 = TransactionUtils.getTransactionFromCaller(signatureCache, FederationChangeCaller.THIRD_AUTHORIZED.getRskAddress());
+
+            ABICallSpec createFederationAbiCallSpec = new ABICallSpec(FederationChangeFunction.CREATE.getKey(), new byte[][]{});
+
+            // Voting with  m of n authorizers to create the pending federation
+            federationSupport.voteFederationChange(tx, createFederationAbiCallSpec, signatureCache, bridgeEventLogger);
+            federationSupport.voteFederationChange(tx2, createFederationAbiCallSpec, signatureCache, bridgeEventLogger);
+
+            Keccak256 pendingFederationHash = federationSupport.getPendingFederationHash();
```

Comment:
I'd add an assertion that the pending federation was created. Could be by calling `getPendingFederationSize` and assert the value is 0. If no pending federation exists then the value is -1

Reply:
The test right before this one is asserting that already. From this line up, these two tests are the same, so, following previous recommendations, I'm trying to avoid asserting what other tests have already asserted. I guess it doesn't hurt adding it either.